frontend/src/metabase-lib/lib/Question.js

```diff
@@ -246,6 +246,25 @@ export default class Question {
     toUnderlyingData(): Question {
         return this.setDisplay("table");
     }
+
+    composeThisQuery(): ?Question {
```

Comment:
It doesn't need to be an instance of a "structured" query (which I assume means an MBQL query?) This PR is so we can use it with native queries